src/common/src/config.rs

```diff
@@ -187,6 +187,9 @@ pub struct ServerConfig {
     /// 0 = close metrics
     /// >0 = open metrics
     pub metrics_level: u32,
+
+    #[serde(default = "default::server::telemetry_enabled")]
+    pub telemetry_enabled: bool,
```

Comment:
I think we can make `telemetry_enabled` as part of the cluster-level system parameter https://github.com/risingwavelabs/risingwave/blob/c4516dea1366d9df06ccf7e28ef2bdabf6fb1019/src/common/src/system_param.rs#L28

This makes it possible for user to enable/disable telemetry in runtime and nodes in the same cluster can have the same `telemetry_enabled` config.

Reply:
Actually, if we also make `tracking_id` as part of the system parameter, we may not need GetTelemetryInfo anymore.

Reply:
IMO, if we move `tracking_id` to system params, it will be configurable while it is not a config field.
 It's a good idea to move `telemetry_enabled` to system params.